src/Index.svelte

```diff
@@ -56,6 +56,7 @@
 
 
 <svg viewBox="0 0 24 24" {style}>
+  <title>{title}</title>
```

Comment:
Maybe this will be better?
```svelte
{#if title}<<title>{title}</title>{/if}
```

Reply:
Good call! Submitting a new commit to address this.